app_spec/test/test.js

```diff
@@ -847,4 +847,13 @@ scenario('request grant', async (s, t, { alice, bob }) => {
     t.equal(result.Ok, grants.Ok[0])
 })
 
+  scenario('emit signal', async (s, t, { alice }) => {
+
+    const result = await alice.callSync("simple", "test_emit_signal", {message: "test message"})
+    // TODO: Here we need some way to read all the received UserSignals...
```

Comment:
@maackle, I've already spotted where Diorama processes signals. Can you point me into a direction where and how to add signals to the object that is `alice` here? I guess that is a `DnaInstance`? How can I access all those from https://github.com/holochain/diorama/blob/65dd181cd72d15813d08272631426be5acf37921/src/diorama.ts#L61.